sstmac/hardware/memory/memory_model.h

```diff
@@ -125,7 +125,7 @@ class MemoryModel : public SubComponent
 
   int initialize(RequestHandlerBase* handler);
 
-  virtual std::string toString() const = 0;
+  std::string toString() const override = 0;
```

Comment:
Same problem. There is no parent virtual method in integrated core. Probably what we should do is completely delete the line here. Then let's make an `SSTIntegratedSubcomponent` that provides the top-level virtual function.